test/spec/unit/pbjs_api_spec.js

```diff
@@ -30,18 +31,19 @@ require('modules/adequantBidAdapter');
 var config = require('test/fixtures/config.json');
 
 $$PREBID_GLOBAL$$ = $$PREBID_GLOBAL$$ || {};
-$$PREBID_GLOBAL$$._bidsRequested = getBidRequests();
-$$PREBID_GLOBAL$$._bidsReceived = getBidResponses();
-$$PREBID_GLOBAL$$.adUnits = getAdUnits();
-$$PREBID_GLOBAL$$._adUnitCodes = $$PREBID_GLOBAL$$.adUnits.map(unit => unit.code);
+var auction = auctionManager.createAuction();
+auction.getBidderRequests = getBidRequests;
+auction.getBidsReceived = getBidResponses;
+auction.getAdUnits = getAdUnits;
+auction.setAdUnitCodes(auction.getAdUnits().map(unit => unit.code));
 
 function resetAuction() {
   $$PREBID_GLOBAL$$.setConfig({ enableSendAllBids: false });
   $$PREBID_GLOBAL$$.clearAuction();
-  $$PREBID_GLOBAL$$._bidsRequested = getBidRequests();
-  $$PREBID_GLOBAL$$._bidsReceived = getBidResponses();
-  $$PREBID_GLOBAL$$.adUnits = getAdUnits();
-  $$PREBID_GLOBAL$$._adUnitCodes = $$PREBID_GLOBAL$$.adUnits.map(unit => unit.code);
+  auction.getBidderRequests = getBidRequests;
+  auction.getBidsReceived = getBidResponses;
+  auction.getAdUnits = getAdUnits;
+  auction.setAdUnitCodes(auction.getAdUnits().map(unit => unit.code));
```

Comment:
Can this also do `auction = auctionManager.createAuction()` now? Cause that'd be awesome :).

If it causes test failures, though, don't worry about it. Many of these tests depend on each others' state in unhealthy ways.